ArcGISRuntimeSDKQt_CppSamples/Layers/BlendRasterLayer/Info.plist

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>RgbRenderer</string>
+	<key>CFBundleExecutable</key>
+	<string>RgbRenderer</string>
+	<key>CFBundleGetInfoString</key>
+	<string>ArcGIS</string>
+	<key>CFBundleIcons~ipad</key>
+	<dict/>
+	<key>CFBundleIdentifier</key>
+	<string>com.esri.${PRODUCT_NAME:rfc1034identifier}</string>
+	<key>CFBundleName</key>
+	<string>${PRODUCT_NAME}</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleSignature</key>
+	<string>????</string>
+	<key>CFBundleVersion</key>
+	<string>1.0</string>
+	<key>NOTE</key>
+	<string>This app is cool</string>
+	<key>UIFileSharingEnabled</key>
+        <string>FALSE</string>
```

Comment:
This should be TRUE. I think I forgot this on the RGB renderer PR. If you could go back and add it, that would be great.